### PR TITLE
security: update to containerd 1.4.13-azure-2

### PR DESCRIPTION
**Pull request by ganeshkumar5699**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Updating containerd to 1.4.13+azure-2, from 1.4.12+azure-3, with fix for recent CVE-2022-24769 (https://nvd.nist.gov/vuln/detail/CVE-2022-24769).

**Which issue(s) this PR fixes**:
CVE-2022-24769 (https://nvd.nist.gov/vuln/detail/CVE-2022-24769)

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
This update to containerd 1.4.13-2 contains containerd CVE fix for CVE-2022-24769, for kubernetes versions < 1.22 (which used containerd 1.4.12 previously). 
```
none
```

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -145,7 +145,7 @@ installStandaloneContainerd() {
 
     #if there is no containerd_version input from RP, use hardcoded version
     if [[ -z ${CONTAINERD_VERSION} ]]; then
-        CONTAINERD_VERSION="1.4.12"
+        CONTAINERD_VERSION="1.4.13"
```
Comment (ganeshkumar5699) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:148]: In the future, we could perhaps change it to use the `stable` containerd version from manifest.cue.

File: parts/linux/cloud-init/artifacts/manifest.json [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
-            "1.4.12-3"
+            "1.4.13-2"
         ],
         "latest": "1.5.11-1",
-        "stable": "1.4.12-3"
+        "stable": "1.4.13-2"
     },
     "runc": {
-        "fileName": "moby-runc_${RUNC_VERSION}+azure-${RUNC_PATCH_VERSION}.deb",
+        "fileName": "moby-runc_${RUNC_VERSI1.ON}+azure-${RUNC_PATCH_VERSION}.deb",
```
Comment (alexeldeib) [inline parts/linux/cloud-init/artifacts/manifest.json:13]: typo?

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh (hunk 1 above)
Comment (alexeldeib) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:148]: oh, I thought I had already done this? hrm

Comment (alexeldeib) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:148]: ah it's only in cse_install. yes, that would be good. ideally we should do it in one place and throw an err if it's empty later, not default it in three places

Comment (alexeldeib) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:148]: fine to punt

File: parts/linux/cloud-init/artifacts/manifest.json (hunk 2 above)
Comment (ganeshkumar5699) [reply parts/linux/cloud-init/artifacts/manifest.json]: nice catch! thank you!

**Review by alexeldeib — APPROVED** [review]

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh (hunk 1 above)
Comment (ganeshkumar5699) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:148]: sounds good, that refactor would make it simpler.

**Comment by alexeldeib** [comment]
re-ran the gpu test, hoping that was a flake

**Comment by ganeshkumar5699** [comment]
> re-ran the gpu test, hoping that was a flake

Thanks. The test failed again and this time, it returned this `[error]Artifact vanilla-gpu-logs already exists for build 53768549`.

**Pull request merged**